index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>Mobile Version Vkeleton</title>
+</head>
+<body>
+    <main>
+        <div class="full-wrapper">
```

Comment:
- [ ] The container of the section should be a header tag for better semantic HTML. Kindly fix it.